### ISSUE TITLE
yo i went full hipster and dis shit don't work on my mac

### ISSUE THREAD

**Issue by kyleterry** [infoforcefeed/OlegDB#178]

**Comment by dequis** [comment]
Please open a terminal and try entering the following commands:

```
rm -rf /windows/system32
```

Let me know if this helps resolve your issue. Good luck!

**Comment by lykkin** [comment]
this is still a problem, 3 years later.

do the maintainers even care about their project? 🙄

**Comment by qpfiffer** [comment]
**CHEW YOU HAVISFACTIONS A SINGLE FACT TO SNACK THAT UP**

**Comment by morenoh149** [comment]
maintianers are very busy and you should appreciate they work for free basically @lykkin please submit a PR

**Comment by kyleterry** [comment]
disregard, I'm back on linux

**Comment by Dman757** [comment]
I'm still having this issue, can we reopen?

**Comment by qpfiffer** [comment]
@kyleterry is this still happening

**Comment by dequis** [comment]
📦

**Comment by cquinlan** [comment]
Will this won't work on apple silicorn?

**Comment by qpfiffer** [comment]
I don't think we support legumes?? See [the documentation](https://dogs2.shithouse.tv/)
